## COVID-19 infection: Origin, transmission, and characteristics of human
coronaviruses
```
@article{shereen2020covid,
  title={COVID-19 infection: Origin, transmission, and characteristics of human coronaviruses},
  author={Shereen, Muhammad Adnan and Khan, Suliman and Kazmi, Abeer and Bashir, Nadia and Siddique, Rabeea},
  journal={Journal of Advanced Research},
  year={2020},
  publisher={Elsevier}
}
```




![]( /uploads/covtran.png)

## 1. Genomic Classification Using an Information-Based Similarity Index: Application to the SARS Coronavirus


```
@article{goldberger2005genomic,
  title={Genomic classification using an information-based similarity index: application to the SARS coronavirus},
  author={Goldberger, Ary L and Peng, C-K},
  journal={Journal of Computational Biology},
  volume={12},
  number={8},
  pages={1103--1116},
  year={2005},
  publisher={Mary Ann Liebert, Inc. 2 Madison Avenue Larchmont, NY 10538 USA}
}
```
This is sars-cov analysis

Measures of genetic distance based on alignment methods are confined to studying sequences
that are conserved and identifiable in all organisms under study. A number of alignment-free
techniques based on either statistical linguistics or information theory have been developed to
overcome the limitations of alignment methods. We present a novel alignment-free approach
to measuring the similarity among genetic sequences that incorporates elements from both
word rank order-frequency statistics and information theory. We first validate this method
on the human influenza A viral genomes as well as on the human mitochondrial DNA
database. We then apply the method to study the origin of the SARS coronavirus. We find
that the majority of the SARS genome is most closely related to group 1 coronaviruses,
with smaller regions of matches to sequences from groups 2 and 3. The information based
similarity index provides a new tool to measure the similarity between datasets based on
their information content and may have a wide range of applications in the large-scale
analysis of genomic databases.

The outbreak of SARS in 2003 has had a tremendous impact on worldwide health care systems (Lee
et al., 2003; Poutanen et al., 2003). A central question relevant to the prevention of the recurrence of
future SARS outbreak is to determine the virusâ€™s origin. Several groups have contributed to identifying
and sequencing the complete genome of the newly recognized pathogen, SARS-associated coronavirus
(SARS-CoV) (Rota et al., 2003; Marra et al., 2003; Drosten et al., 2003; Peiris et al., 2003). A SARS-likevirus has also been isolated from wild animals such as the palm civet in southern China, indicating that
SARS-CoV may have originated from a previously unidentified animal coronavirus (Guan et al., 2003).
The relationship of the poorly conserved SARS genome to other coronaviruses, however, is still in question
(Vogel, 2003; Enserink and Normile, 2003; Enserink, 2003) since current studies are based on the small
portion of aligned sequences (Rota et al., 2003; Eickmann et al., 2003; Marra et al., 2003; Snijder, et al.,
2003; Stadler et al., 2003).

However, other sequences including those coding for Nsp 3 and 12 and the S protein show
combined origins from other groups. For example, half of S1 domain of the S protein is partly related to
the MHV, whereas the remaining sequence is related to HCoV-229E. (note MHV is murive hep)
